ddtrace/contrib/botocore/patch.py

```diff
@@ -71,4 +73,8 @@ def patched_api_call(original_func, instance, args, kwargs):
         span.set_tag(http.STATUS_CODE, result['ResponseMetadata']['HTTPStatusCode'])
         span.set_tag("retry_attempts", result['ResponseMetadata']['RetryAttempts'])
 
+        analytics_sample_rate = config.botocore.get_analytics_sample_rate(use_global_config=False)
```

Comment:
We should probably reverse the default for `use_global_config`.

Have it be `False` by default and override for the specific integrations we want to inherit from it.

The reason for this is if we ever want to change the default from `False` to `True` (all integrations inherit by default) then we'll have to go through everywhere and update this arg.

Reply:
I don't have a strong opinion here, so if you think `True` is the better default then :+1: